HACKING.md

```diff
@@ -17,4 +19,42 @@ dfx start --clean --background
 dfx deploy
 ```
 
-You can now visit the frontend in your browser and sign messages!
+You can now visit the front end in your browser and sign messages!
+
+## GitHub Release
+
+Releases are created by a GitHub action.
+
+For a test release, just push any tag and a release will be created for that tag.
+
+For a production release:
+
+- Create a release branch.
+- Update the version numbers in the git repository, with: `./scripts/version-bump [patch|minor|major|alpha|beta|rc]` (default: patch)
+- Merge the release branch.
+- Tag the merged code with: `scripts/release`.
+  - Note: This will create a tag and push it to GitHub. A GitHub action will then create a release.
```

Comment:
Github creates a "Draft" release. We then need to publish it from the UI.

Should we add this or change the job to also publish it?

Reply:
Good point.  Given that the release process is new, I think it would be good if people sanity checked the release prior to publishing and maybe added release notes.  I'll add this to the instructions.